packages/jest/src/index.mjs

```diff
@@ -5,41 +5,54 @@ import globals from 'globals';
 /**
  * @type {import('eslint').Linter.Config[]}
  */
-const config = createConfig({
-  name: '@metamask/eslint-config-jest',
+const config = createConfig([
+  {
+    name: '@metamask/eslint-config-jest',
 
-  extends: [jest.configs['flat/recommended'], jest.configs['flat/style']],
+    extends: [jest.configs['flat/recommended'], jest.configs['flat/style']],
 
-  languageOptions: {
-    globals: {
-      ...globals.jest,
+    languageOptions: {
```

Comment:
These changes are just indentation, I haven't changed any actual rules here. The only real changes in this file are on the bottom.